.gitignore

```diff
@@ -18,6 +18,7 @@
 # Ignore the default SQLite database.
 /db/*.sqlite3
 /db/*.sqlite3-journal
+/db/structure.sql
```

Comment:
This is not acceptable. The db/structure.sql is a fundamental part of the application and cannot be `gitignore`d without creating problems in the near future.